### PR TITLE
Update Oracle DML SLELECT PERCENT_RANK statement parse

### LINKED ISSUE #22041: Cannot parse Oracle DML SELECT PERCENT_RANK statement
### Which version of ShardingSphere did you use?
7ca33ec7df4e5f110f1b7acd441b8e7439eb401e
### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?
SQL parser
### Expected behavior
parse SELECT PERCENT_RANK statement
### Actual behavior
cannot parse SELECT PERCENT_RANK statement
### Steps to reproduce the behavior, such as: SQL to execute, sharding rule configuration, when exception occur etc.
```sql
SELECT PERCENT_RANK(15000, .05) WITHIN GROUP (ORDER BY salary, commission_pct) "Percent-Rank"   FROM employees;
```

### PR DESCRIPTION
**Pull request by Qianyi951015**
Fixes #22041

Ref: https://docs.oracle.com/en/database/oracle/oracle-database/12.2/sqlrf/PERCENT_RANK.html#GUID-66A868F5-9EBA-482A-BF8C-09300B9EE165

image

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [x] I have passed maven check locally : `mvn clean install -B -T2C -DskipTests -Dmaven.javadoc.skip=true -e`.
- [ ] I have made corresponding changes to the documentation.
- [x] I have added corresponding unit tests for my changes.

### CODE REVIEW

**Comment by RaigorJiang** [comment]
@Qianyi951015 Thank you, I will check the PR.

File: sql-parser/dialect/oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/statement/impl/OracleStatementSQLVisitor.java [hunk 1]
```diff
@@ -519,7 +519,7 @@ private ASTNode createAggregationSegment(final AggregationFunctionContext ctx, f
     }
     
     private Collection<ExpressionSegment> getExpressions(final AggregationFunctionContext ctx) {
-        return null == ctx.expr() ? Collections.emptyList() : Collections.singletonList((ExpressionSegment) visit(ctx.expr()));
+        return null == ctx.exprs() ? Collections.emptyList() : Collections.singletonList((ExpressionSegment) visit(ctx.exprs()));
```
Comment (RaigorJiang) [inline sql-parser/dialect/oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/statement/impl/OracleStatementSQLVisitor.java:522]: Hi @Qianyi951015 , I found that there is not method to visit ExprsContext, so this line will not give the expected result.

Comment (Qianyi951015) [reply sql-parser/dialect/oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/statement/impl/OracleStatementSQLVisitor.java]: I have corrected it and pushed again

Comment (RaigorJiang) [reply sql-parser/dialect/oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/statement/impl/OracleStatementSQLVisitor.java]: OK, thank you！

**Comment by RaigorJiang** [comment]
Hi @tuichenchuxin , could you please help to review this PR, thanks!

**Review by tuichenchuxin — APPROVED** [review]

**Pull request merged**